src/libraries/System.Net.Http/src/System/Net/Http/Headers/EntityTagHeaderValue.cs

```diff
@@ -35,56 +36,31 @@ public EntityTagHeaderValue(string tag)
 
         public EntityTagHeaderValue(string tag, bool isWeak)
         {
-            ArgumentException.ThrowIfNullOrWhiteSpace(tag);
+            HeaderUtilities.CheckValidQuotedString(tag);
```

Comment:
Now, this one is going to throw different messsage. But I think this is intentional right?

Reply:
Yes, `null` would behave the same, but empty/whitespace values will now go back to throwing `FormatException` instead of `ArgumentException` (like they did before #86007).